cfy_manager/components/prometheus/prometheus.py

```diff
@@ -0,0 +1,357 @@
+                           ignore_failure=True)
+        service.remove(NODE_EXPORTER, append_prefix=False)
+        service.remove(BLACKBOX_EXPORTER, append_prefix=False)
+        service.remove(POSTGRES_EXPORTER, append_prefix=False)
+        service.remove(RABBITMQ_EXPORTER, append_prefix=False)
+        service.remove(PROMETHEUS, append_prefix=False)
+        logger.notice('Successfully removed Prometheus and exporters files')
+
+    def start(self):
+        logger.notice('Starting Prometheus and exporters...')
+        service.restart(PROMETHEUS, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS]['port'])
+        service.restart(NODE_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS][NODE_EXPORTER]['metrics_port'])
+        service.restart(BLACKBOX_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS][BLACKBOX_EXPORTER]['metrics_port'])
+        service.restart(POSTGRES_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS][POSTGRES_EXPORTER]['metrics_port'])
+        service.restart(RABBITMQ_EXPORTER, append_prefix=False,
+                        ignore_failure=True)
+        # wait_for_port(config[PROMETHEUS][RABBITMQ_EXPORTER]['metrics_port'])
+        _validate_prometheus_running()
+        logger.notice('Prometheus and exporters successfully started')
+
+    def stop(self):
+        logger.notice('Stopping Prometheus and exporters...')
+        service.stop(PROMETHEUS, append_prefix=False)
+        service.stop(NODE_EXPORTER, append_prefix=False)
+        service.stop(BLACKBOX_EXPORTER, append_prefix=False)
+        service.stop(POSTGRES_EXPORTER, append_prefix=False)
+        service.stop(RABBITMQ_EXPORTER, append_prefix=False)
+        logger.notice('Prometheus and exporters successfully stopped')
+
+    def join_cluster(self, join_node):  # , restore_users_on_fail=False):
+        logger.info(
+            'Would be joining cluster via node {target_node}.'.format(
+                target_node=join_node,
+            )
+        )
+
+
+def _create_directory(directory, use_sudo=True):
+    common.mkdir(directory, use_sudo=use_sudo)
+    return directory
+
+
+def _download_release(dest_dir, base_url, name=None, version=None):
+    if name and version:
+        logger.notice(
+            'Downloading {0}-{1} to {2}'.format(name, version, dest_dir))
+        tarball_url = '{0}/{1}/releases/download/v{2}/{1}-{2}.{3}'.format(
+            base_url, name, version, 'linux-amd64.tar.gz')
+    else:
+        logger.notice('Downloading {0} to {1}'.format(base_url, dest_dir))
+        tarball_url = base_url
+    archive_file_name = join(dest_dir, '{0}-{1}.tar.gz'.format(name, version))
+    common.run(['curl', '-L', '-o', archive_file_name, tarball_url])
+    return archive_file_name
```

Comment:
@mateumann I think we should provide this as part of rpm instead of downloading them directly from internet as some of our customers does not support internet connection. for example check this example here https://github.com/cloudify-cosmo/cloudify-versions/blob/master/packages-urls/manager-packages.yaml#L23

Reply:
Right, I'll add `*.spec`s.

Reply:
@mateumann I do not think so that we handle the archices as spec, maybe its worth to check how the sanity hello world app get handled and do the same for exporters

Reply:
I've added the `*.spec` files.  Those are used in CircleCI job to create five RPMs `build_prometheus_rpms`, which are then packed inside `cloudify-manager-install-*.rpm`
Next step could be to create relevant Jenkins jobs to create RPMs separately (and upload somewhere else than just CircleCI workspace)